sips/sip-x/sip-x-nft-metadata.md

```diff
@@ -0,0 +1,389 @@
+| `raw_media_file_signature`      | `string`  | signature of the media file represented by the token                                                                                                                                                                                                                                                                                                                |
+| `raw_media_file_signature_type` | `string`  | signature type of the media represented by the token, e.g. SHA-256                                                                                                                                                                                                                                                                                                  |
+| `animation_url`                 | `string`  | url to a multi-media attachment for the item. Application might use this to display the token in a richer way than the image of the `image` property. Application might support media types like GLTF, GLB, WEBM, MP4, M4V, OGV, and OGG, MP3, WAV, and OGA as well as HTML. The query `?ext={file_extension}` can be used to provide information on the file type. |
+| `image_data`                    | `string`  | raw SVG image data.                                                                                                                                                                                                                                                                                                                                                 |
+| `category`                      | `string`  | category of the associated media file, e.g. `image`, `video`, `audio`, `vr`, `html`.                                                                                                                                                                                                                                                                                |
+| `files`                         | `array`   | list of all associated files, represented as `{uri: string, type: string, signature: string, signature_type: string}`.                                                                                                                                                                                                                                              |
+| `creators`                      | `array`   | list of creators and their shares, represented as `{address: string, share: number}`.                                                                                                                                                                                                                                                                               |
```

Comment:
How is "share" specified?  as a uint?  as a percentage?  Is the context for royalties?  Should royalties be speciifed separately?

Reply:
clarification has been added